app/controllers/api/v2/sales_controller.rb

```diff
@@ -3,7 +3,7 @@
 class Api::V2::SalesController < Api::V2::BaseController
   before_action(only: [:index, :show]) { doorkeeper_authorize! :view_sales }
   before_action(only: [:mark_as_shipped]) { doorkeeper_authorize! :mark_sales_as_shipped }
-  before_action(only: [:refund]) { doorkeeper_authorize! :refund_sales }
+  before_action(only: [:refund, :resend_receipt]) { doorkeeper_authorize! :edit_sales }
```

Comment:
Doorkeeper scopes are stored on the token, so this won't work unless we have a strategy to migrate (large numbers of) existing tokens

Since we have a separate scope for `mark_sales_as_shipped` I think we should create a new `resend_receipts` scope instead and update existing `OauthApplication` records to include it (there are much fewer of those than tokens)

@ershad lmk if you disagree!